news/5572.feature.rst

```diff
@@ -0,0 +1 @@
+Implement PIP_TRUSTED_HOSTS logic from importing requirements to specified --index URL.
```

Comment:
Can you please make the news snippet more informative? Maybe with an example usage?
I feel like this would be very helpful to make this change more clear.

Reply:
Sure! Always open to learning how to do this correctly.

Reply:
Let me know if the change is good.